src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java

```diff
@@ -114,13 +139,12 @@ public Ship placeEnemyShip(String name, int length){
         while(!valid) {
             //select a random orientation; 1 == vertical; 2 == horizontal
             //random Coordinate for start point
-            orientation = (Math.random() <= 0.5) ? 1 : 2;
-            row = (int) (Math.random() * BOARD_SIZE) + 1;
-            col = (int) (Math.random() * BOARD_SIZE) + 1;
+            orientation = enemyShipOrientation(isEasy,orientation);
+            row = enemyShipRow(isEasy,orientation); // sets row using enemyShipRow function
+            col = enemyShipColumn(isEasy,orientation); // sets col using enemyShipColumn function
```

Comment:
By using these methods and passing isEasy, the ships will be placed correctly according to the game difficulty. Nice job.